src/bp-groups/bp-groups-template.php

```diff
@@ -3470,7 +3470,7 @@ function bp_get_group_member_promote_mod_link( $args = '' ) {
 		$args,
 		array(
 			'user_id' => $members_template->member->user_id,
-			'group'   => &$groups_template->group,
+			'group'   => ! empty( $groups_template->group ) ? $groups_template->group : groups_get_current_group(),
```

Comment:
please revert this and whenever it's required please use the args of the function.